app/controllers/benefits/fuel/CompanyCarBenefitsController.scala

```diff
@@ -33,8 +32,10 @@ import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendController
 import utils.{Clock, SessionHelper}
 import views.html.benefits.fuel.CompanyCarBenefitsView
 
+import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
+// TODO: This controller does not have enough test coverage in CompanyCarBenefitsControllerISpec
```

Comment:
is this still true? or did you add tests to up the coverage?

Reply:
Still true. I did not add tests. This story is already quite big. Did not want to make it bigger.

Reply:
fair enough